l10n_ro_account_edi_ubl/models/account_edi_format.py

```diff
@@ -46,6 +46,26 @@ def _get_xml_builder(self, company):
             return self.env["account.edi.xml.cius_ro"]
         return super()._get_xml_builder(company)
 
+    def _find_value(self, xpath, xml_element, namespaces=None):
+        res = None
+        try:
+            res = super(AccountEdiXmlCIUSRO, self)._find_value(
+                xpath, xml_element, namespaces=namespaces
+            )
+        except Exception:
+            namespaces = {
+                "qdt": "urn:oasis:names:specification:ubl:schema:xsd:QualifiedDataTypes-2",
+                "ccts": "urn:un:unece:uncefact:documentation:2",
+                "udt": "urn:oasis:names:specification:ubl:schema:xsd:UnqualifiedDataTypes-2",
+                "cac": "urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2",  # noqa: B950
+                "cbc": "urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2",
+                "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+            }
+            res = super(AccountEdiXmlCIUSRO, self)._find_value(
+                xpath, xml_element, namespaces=namespaces
+            )
+        return res
+
```

Comment:
nu e mai ok     ceva de genul ?

def _find_value(self, xpath, xml_element, namespaces=None):
        if namespaces is None:
            namespaces = {}

        namespaces.update(
            {
                "qdt": "urn:oasis:names:specification:ubl:schema:xsd:QualifiedDataTypes-2",
                "ccts": "urn:un:unece:uncefact:documentation:2",
                "udt": "urn:oasis:names:specification:ubl:schema:xsd:UnqualifiedDataTypes-2",
                "cac": "urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2",  # noqa: B950
                "cbc": "urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2",
                "xsi": "http://www.w3.org/2001/XMLSchema-instance",
            }
        )

        res = super()._find_value(xpath, xml_element, namespaces=namespaces)

        return res